backend/src/btrgb.py

```diff
@@ -0,0 +1,78 @@
+""" btrgb.py
+Generating .btrgb output files.
+
+Functions:
+
+Authors:
+    Sungmin Kim
+
+License:
+    © 2022 BeyondRGB
+    This code is licensed under the MIT license (see LICENSE.txt for details)
+"""
+import json
+from lab_refs import LAB_REF
+import numpy as np
+
+class Btrgb:
+    """
+    Class representing values to be output to btrgb file.
+
+    Members:
+        OutPutFiles              : List of files to be outputted
+        CalibrationResults       : Dict of calibration values such as DeltaE values
+        VerificationResults      : Dict of verification values
+
+    Methods: 
+        create_output_files         : Set output files
+        create_calibration_results  : Set calibration results
+        create_verification_results : Set verification results
+        to_json                  : Helper function returning a json object representing a btrgb class
+        write_to_file            : Write btrgb class to a file in json format
+    """
+
+    OutPutFiles = {}
+    CalibrationResults = {}
+    VerificationResults = {}
+
+    def __init__(self):
+        self.CalibrationResults = {
+            "double_values": [None],
+            "matrix_values": [None for _ in range(7)]
```

Comment:
If this is just an array of Nones, it could probably just be  `[None, None, None, None, None, None, None]` to make it more explicit - but this isn't super important

Reply:
You can also do "[None] * 7"